DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/README.md

```diff
@@ -0,0 +1,356 @@
+  >  ```
+  >  cmake -G "NMake Makefiles" .. -DFPGA_DEVICE=<board-support-package>:<board-variant> -DIS_BSP=1
+  >  ```
+  > **Note**: You can poll your system for available BSPs using the `aoc -list-boards` command. The board list that is printed out will be of the form
+  > ```
+  > $> aoc -list-boards
+  > Board list:
+  >   <board-variant>
+  >      Board Package: <path/to/board/package>/board-support-package
+  >   <board-variant2>
+  >      Board Package: <path/to/board/package>/board-support-package
+  > ```
+  >
+  > You will only be able to run an executable on the FPGA if you specified a BSP.
+
+3. Compile the design. (The provided targets match the recommended development flow.)
+
+   1. Compile for emulation (fast compile time, targets emulated FPGA device).
+      ```
+      nmake fpga_emu
+      ```
+   2. Compile for simulation (fast compile time, targets simulator FPGA device):
+      ```
+      nmake fpga_sim
+      ```
+   3. Generate HTML performance report.
+      ```
+      nmake report
+      ```
+      The report resides at `svd_report.a.prj/reports/report.html`.
+
+   4. Compile for FPGA hardware (longer compile time, targets FPGA device).
+      ```
+      nmake fpga
+      ```
+>**Note**: If you encounter any issues with long paths when compiling under Windows*, you may have to create your 'build' directory in a shorter path, for example `C:\samples\build`. You can then run cmake from that directory, and provide cmake with the full path to your sample directory, for example:
+>
+>  ```
+  > C:\samples\build> cmake -G "NMake Makefiles" C:\long\path\to\code\sample\CMakeLists.txt
+>  ```
+## Run the `SVD` Design
+
+### Configurable Parameters
+
+| Argument  | Description
+|:---       |:---
+| `<num>`   | (Optional) Specifies the number of times to repeat the decomposition. Its default value is **1** for the emulation and simulation flow and **16384** for the FPGA flow.
+
+### On Linux
+
+#### Run on FPGA Emulator
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
+   ```
+   ./svd.fpga_emu
+   ```
+
+#### Run on FPGA Simulator
+
+1. Run the sample on the FPGA simulator.
+   ```
+   CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./svd.fpga_sim
+   ```
+
+#### Run on FPGA
+
+1. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`).
+   ```
+   ./svd.fpga
+   ```
+
+### On Windows
+
+#### Run on FPGA Emulator
+
+1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
+   ```
+   svd.fpga_emu.exe
+   ```
+
+#### Run on FPGA Simulator
+
+1. Run the sample on the FPGA simulator.
+   ```
+   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
+   svd.fpga_sim.exe
+   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=
+   ```
+
+## Example Output
+
+Example Output when running on the **Terasic DE10-Agilex Development Board**.
+
+```
+Running on device: de10_agilex : Agilex Reference Platform (aclde10_agilex0)
```

Comment:
Run this sample on N6001 to update the readme from DE10 to N6001 as we moved to this board.